app/frontend/src/features/messages/messagelist/MessageList.tsx

```diff
@@ -46,9 +46,12 @@ export default function MessageList({
 
       <>
         {messages.length ? (
-          messages.map((message) => (
-            <MessageView key={message.messageId} message={message} />
-          ))
+          messages
+            .slice()
+            .reverse()
+            .map((message) => (
+              <MessageView key={message.messageId} message={message} />
+            ))
```

Comment:
Rather than reversing the message list in JavaScript, I found that you can achieve this using CSS alone if you set `Box` to also be a flex container (setting `display` to `flex`) and have `flexDirection` set to `column-reverse`.

Reply:
Fixed